files/external_node_v2.rb

```diff
@@ -196,8 +196,10 @@ def enc(certname)
   end
   res = http.start { |http| http.request(req) }
 
-  raise "Error retrieving node #{certname}: #{res.class}\nCheck Foreman's /var/log/foreman/production.log for more information." unless res.code == "200"
-  res.body
+  return res.body if res.code == '200'
+
+  raise res.class, "Error retrieving node #{certname}: #{res.class}"\
```

Comment:
Sorry!  That was a `Net::HTTPInternalServerError` (the first I was testing with).